packages/web3-eth-ens/src/lib/ResolverMethodHandler.js

```diff
@@ -112,18 +124,23 @@ ResolverMethodHandler.prototype.send = function (sendOptions, callback) {
  */
 ResolverMethodHandler.prototype.handleCall = function (promiEvent, method, preparedArguments, callback) {
     method.apply(this, preparedArguments).call()
-        .then(function (receipt) {
-            promiEvent.resolve(receipt);
-
+        .then(function (result) {
             if (_.isFunction(callback)) {
-                callback(receipt);
+                // It's required to pass the receipt to the second argument to be backwards compatible and to have the required consistency
+                callback(result, result);
```

Comment:
Just double-checking but basically you've changed the callback format here to something more standard?

e.g from
```
callback(error|receipt)
```
to
```
callback(error, receipt)
```

Reply:
Yes, I've increased the consistency as described by preventing backward compatibility.

See PR desc:
> I've updated the callback signature to be aligned with the rest of the lib but with respecting of backward compatibility